modules/generate-static-network-policies.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: PROCEDURE
+[id="generate-static-network-policies_{context}"]
+= Generating static network policies
+
+[role="_abstract"]
+The static network policy generator uses link:https://np-guard.github.io/[NP-Guard], which allows you to create and support Kubernetes and {ocp} network policies for zero-trust environments.
+
+[IMPORTANT]
+====
+Using static network policy generator requires you to download and compile `roxctl` command-line interface (CLI).
+====
+
+.Prerequisites
+* Refer to the link:https://github.com/stackrox/stackrox#build-tooling[Build tooling] section of the StackRox repository.
+
+.Procedure
+. Clone the StackRox repository locally and open the `stackrox` directory.
++
+[source,terminal]
+----
+$ git https://github.com/stackrox/stackrox.git && cd stackrox
+----
+. Generate the `roxctl` CLI binary for your operating system:
```

Comment:
"Generate" is okay, but we are actually compiling it in this step. Could we maybe use verbs like: "build", or "compile"?

Reply:
Thank you @vikin91 I'll update.